<launch>
    <arg name="folder_path" default="$(find sandbox)/waypoints"/>
    <arg name="file_name"/>
    <param name="/waypoint_file_path" value="$(arg folder_path)/$(arg file_name)"/>
</launch>
